packages/react-ui-core/src/Photo/BackgroundPhoto.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['BackgroundPhoto'], {
+  pure: true,
+})
+
+export default class BackgroundPhoto extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    fallbackUrl: PropTypes.string,
```

Comment:
Looking at the definition of `background-image` style, it seems that when you specify multiple URLs it layers one on top of the other: [Using_multiple_backgrounds](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds)

So the term "fallback" seems a little misleading, and could lead to some unintended problems:
* If the image and the fallback image are different sizes, the fallback image might show up behind the main image.
* If the main image contains transparency, the fallback image might show through the transparent portions of the main image
* Both the main image and the fallback image will always be loaded and rendered to the page by the browser

Considering all the above it might be worthwhile adding some comments / warnings, to ensure this is used only in situations where it will not cause problems.

Reply:
Yeah, I hear you. The reason this is called `fallbackUrl` was to match with the Photo component. That way you don't have to do different things to essentially pass the same logic to each of them.